.github/workflows/TLS-test.yml

```diff
@@ -20,12 +20,13 @@ jobs:
         uses: actions/setup-java@v2
         with:
           java-version: '8'
-          distribution: 'adopt'
+          distribution: 'zulu'
```

Comment:
Have you tried all major distributions?

Reply:
well, I have tried OpenJDK, Oracle JDK, and Zulu.